vitrina/datasets/templates/vitrina/datasets/detail.html

```diff
@@ -22,11 +22,6 @@
 
 {% block content %}
```

Comment:
Add dataset contextual navigation menu (tabs), similar to the one added to organization details template.